### ISSUE TITLE
Create separate “student” and “teacher” portals for projects

### ISSUE THREAD

**Issue by andylolz** [CodeClub/ideas#4]
Definitely not a new suggestion! @CodeClubRik and I have discussed it before, but I’m not sure it’s written down on github anywhere. This is really `codeclub/lesson_format`-related, but I’m putting it here because it represents a significant change.

The projects site is currently quite “teacher-facing”, and therefore not ideal to point students to directly (I generally create a bitly link to the project itself). It would be great to generate separate menu pages that are more student-facing.

**Comment by CodeClubRik** [comment]
> Whilst we're on this...

> If there were a checklist for each activity, then flaging up that the resources are either available in Scratch, or have to be downloaded - would be useful too... so including the projects.codeclub.org.uk URL might be helpful too..

> Sorry - that's off topic

Steven Flower posted the above in the Scratch repo (in another issue) and I'm adding it in here because I think it's relevant. 

I think before we can give students a link in the notes to the downloadable project resources, we probably need to split them up from the project solution. Currently each project's downloadable materials has a 'Project Resources' folder which they should be able to download, and a 'Club Leader Resources' folder which they shouldn't.

**Comment by andylolz** [comment]
> […] adding it in here because I think it's relevant.

Agree, yes! Thanks for adding it here.

> […] which they should be able to download, and a 'Club Leader Resources' folder which they shouldn't.

Great suggestion. Totally agree

**Comment by martinpeck** [comment]
Having discussed with @rikcross we're going to close this as the new site is not teacher-facing (at least, not as much as before).
